### PR TITLE
Feat/add nibiru as a native chain

### PR DESCRIPTION
**Pull request by cgilbe27**
Add nibiru as a native chain
Fix lint error

### CODE REVIEW

**Comment by editaahn** [comment]
Hi! Let me give you some requests on it.
- Some files are managed by Keplr team. Please remove the changes of the files below.
  - src/validate
  - src/constants
- There are rules for file and directory names to be relevant to the chain identifier. Please check the references in its README.

Make sure every validation is successfully done by Github Action. You could see the details on the README.

**Comment by editaahn** [comment]
Sorry for late response.

1. Hmm it's eslint error and seems weird since all the PRs on this repo have gotten accepted so far. You could keep the change. Thanks.
2. If actual chain id is `cataclysm`, please update the `chainId` field and directory/file names to follow it.

File: images/nibiru/chain.png
Comment (editaahn) [inline images/nibiru/chain.png:1]: Additionally, every chain info json needs its own directory and image following the chain identifier.
ex. nibiru-testnet.json file needs images/nibiru-testnet/chain.png

File: cosmos/nibiru-devnet-2.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "chainId": "nibiru-devnet-2",
```
Comment (editaahn) [inline cosmos/nibiru-devnet-2.json:2]: We can't handle nibiru-devnet-1, nibiru-devnet-2 as different chains.
In turn, you should register only one chain information per a chain identifier. If you're confused about the term, could you please check the explanation about chain identifier on README one more time?

Comment (editaahn) [reply cosmos/nibiru-devnet-2.json:2]: https://github.com/chainapsis/keplr-chain-registry?tab=readme-ov-file#chain-registration-directory-structure

**Comment by cgilbe27** [comment]
Updated prettier because of: 
image

**Comment by cgilbe27** [comment]
Addressed feedback @editaahn

**Comment by editaahn** [comment]
Hey @cgilbe27, I removed the change on native chain identifiers. It isn't considered as a native chain firmly, so if I get a relevant request from our partnership team, then I'll be taking on it. Thank you!

**Pull request merged**